src/lib/config.js

```diff
@@ -17,7 +17,6 @@ module.exports = {
   HANDLERS_CRON: RC.HANDLERS.CRON,
   HANDLERS_CRON_DISABLED: RC.HANDLERS.CRON.DISABLED,
   HANDLERS_CRON_TIMEXP: RC.HANDLERS.CRON.TIMEXP,
-  HANDLERS_CRON_TIMEZONE: RC.HANDLERS.CRON.TIMEZONE,
```

Comment:
Its still there in default.json file
```
    "CRON": {
      "DISABLED": false,
      "TIMEXP": "*/10 * * * * *",
      "TIMEZONE": "UTC"
    },
```

In addition, would like to understand both CRON and TIMEOUT values, specifically what the TIMEXP and TIMEZONE on both mean (if there's any duplication)

Reply:
@vgenev I dont think we use:
```
  HANDLERS_CRON: RC.HANDLERS.CRON,
  HANDLERS_CRON_DISABLED: RC.HANDLERS.CRON.DISABLED,
  HANDLERS_CRON_TIMEXP: RC.HANDLERS.CRON.TIMEXP,
```

Thus that and the config from default.json should be removed:

```
  "HANDLERS": {
   ....,
    "CRON": {
      "DISABLED": false,
      "TIMEXP": "*/10 * * * * *",
      "TIMEZONE": "UTC"
    },
```

Please confirm if this is correct, and if so, then remove.

Reply:
removed